nginx/html/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <title>TNO - Today's News Online</title>
+  <style>
```

Comment:
Just noticed the TNO in the title, probably doesn't matter, but would this file be used at anytime in PIMS?

Reply:
I copied it from TNO.  Needs to be updated for PIMS.  This is the maintenance page.  It should at some point be updated to look better.  But I don't have time.